src/Utils/Hooks/useAuthIntent/index.tsx

```diff
@@ -16,6 +16,7 @@ export const AFTER_AUTH_ACTION_KEY = "afterSignUpAction"
 
 export type AfterAuthAction =
   | { action: "associateSubmission"; kind: "submission"; objectId: string }
+  | { action: "createAlert" }
   | { action: "createAlert"; kind: "artist"; objectId: string }
   | { action: "createAlert"; kind: "artworks"; objectId: string }
```

Comment:
I'm fairly certain that we no longer need these `kind` / `objectId` attributes. The page sets up the `initialCriteria` on load and all we need to do is dispatch the modal.

Reply:
It seems like we need the `kind` / `objectId` attributes. If they are not set on the artwork page, the modal doesn't open.

I've added this [commit](https://github.com/artsy/force/pull/13129/commits/a7f56cdc35430d2059ca0e020a219ca861152530) to the PR that makes the adjustment. But I'm happy to look into this together because I don't fully understand how we dispatch the modal.

Reply:
Hmmm, yeah - happy to take a look at this with you today 👍

Reply:
Actually, it seems to work now without kind / objectId attributes and I cannot reproduce the issue. Maybe I did something else wrong. I'll merge the changes